internal/net/call/call.go

```diff
@@ -43,14 +43,14 @@ package call
 // messages from the connection.
 //
 // When the client wants to send an RPC, it selects one of its server
-// connections to use, creates a call component, assigns it a new request-id, and
-// registers the components in a map in the connection. It then sends a request
-// message over the connection and waits for the call component to be marked as
+// connections to use, creates a call object, assigns it a new request-id, and
+// registers the object in a map in the connection. It then sends a request
+// message over the connection and waits for the call object to be marked as
 // done.
 //
 // When the response arrives, it is picked up by readResponses().
```

Comment:
I think some of the comments on `readResponses()` are stale now.

Reply:
Thanks for catching that.